src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/settings/SimulationSettingsWindow.java

```diff
@@ -1,45 +1,135 @@
 package edu.kit.pse.osip.simulation.view.settings;
 
 import edu.kit.pse.osip.core.io.files.ServerSettingsWrapper;
-import edu.kit.pse.osip.core.model.base.ProductionSite;
 import edu.kit.pse.osip.core.model.base.TankSelector;
 import edu.kit.pse.osip.core.utils.formatting.InvalidPortException;
+import edu.kit.pse.osip.core.utils.language.Translator;
 import edu.kit.pse.osip.simulation.controller.SettingsChangedListener;
+import edu.kit.pse.osip.simulation.controller.SimulationSettingsInterface;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * This class is the main window of the OSIP simulation settings.
  */
-public class SimulationSettingsWindow  extends javafx.stage.Stage implements edu.kit.pse.osip.simulation.controller.SimulationSettingsWindow {
-    private PortTextField portField;
+public class SimulationSettingsWindow implements SimulationSettingsInterface {
+    private ServerSettingsWrapper settings;
+    private Stage settingsStage;
+    private Map<TankSelector, PortTextField> portTextFields = new HashMap<>();
```

Comment:
Ich denke, es ist besser, eine EnumMap hier zu nehmen. Außerdem verlege bitte die Initialisierung in den Konstruktor.